server/controllers/shoppingList.js

```diff
@@ -8,7 +8,8 @@ const createNewList = (req, resp) => {
   const shoppingList = new ShoppingList({
     description,
     name,
-    adminIds: adminId
+    adminIds: adminId,
+    isArchived: false
```

Comment:
you can set this property in data model using 
isArchived: {type: String, default: false}